src/server/android/android.ts

```diff
@@ -173,8 +212,20 @@ export class AndroidDevice {
       await new Promise(f => setTimeout(f, 100));
     }
     debug('pw:android')('Got the socket, connecting');
-    const androidBrowser = new AndroidBrowser(this, packageName, socketName);
+    return await this._connectToBrowser(socketName, options);
+  }
+
+  connectToWebView(pid: number): Promise<BrowserContext> {
+    const webView = this._webViews.get(pid);
+    if (!webView)
+      throw new Error('WebView with given pid does not exist');
```

Comment:
Let's throw "WebView has been closed", because that's what will happen in real scenarios.